docs/Manual/Tutorials/Starter/README.md

```diff
@@ -87,10 +87,42 @@ docker run -it --name=adb1 --rm -p 8528:8528 \
     --starter.address=$IP
 ```
 
+
 The executable will show the commands needed to run the other instances.
 
 Note that the commands above create a docker volume. If you're running on Linux
-it is also possible to use a host mapped volume. Make sure to map it on `/data`.
+it is also possible to use a host mapped volume. Make sure to map it
+on `/data`.
+
+**TLS verified Docker services**
+
+Oftentimes, one needs to harden Docker services using client certificate 
+and TLS verification. The Docker API allows subsequently only
+certified access. As the ArangoDB starter starts the ArangoDB cluster
+instances using this Docker API, it is mandatory that the ArangoDB
+starter is deployed with the proper certificates handed to it, so that
+the above command is modified as follows: 
+
+```bash
+export IP=<IP of docker host>
+export DOCKER_TLS_VERIFY=1
+export DOCKER_CERT_PATH=/path/to/certificate
+docker volume create arangodb
+docker run -it --name=adb --rm -p 8528:8528 \
```

Comment:
How does the starter know where the docker CA certificate is?
I guess you have to add a `-e DOCKER_CERT_PATH=$(DOCKER_CERT_PATH)` to the docker cmdline.

Reply:
I documented this. 
You only have to specify, if it's not in `$HOME/.docker`

Reply:
the `export..` line only adds it to the `docker run` process, not into the container started by `docker run`.